commcare_connect/templates/prelogin/contact.html

```
@@ -107,8 +107,8 @@
           <a href="{% url 'prelogin:portfolio' %}">Portfolio</a>
           <a href="{% url 'prelogin:frontline-network' %}">Frontline Network</a>
           <div class="nav-cta">
-            <a href="{{ app_login_url }}" class="btn btn-ghost">Sign In</a>
-            <a href="{% url 'account_signup' %}" class="btn btn-primary">Sign Up</a>
+            <a href="{% url "account_login" %}" class="btn btn-ghost">Sign In</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)